### PR TITLE
Feature/nicps 23

### PR DESCRIPTION
**Pull request by ajinkya81**
This ticket is create a front end experience for displaying login history. 
This page should appear after a user logs in and displays some login history.
The page  contain two links:
1. Proceed to Dashboard Link will take to the user to their mail box 
2. View Details Link is intended to allow the end user to see more of their login history.

### CODE REVIEW

File: WebRoot/public/login_history.jsp [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<style>
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: Please add directives of buffer, session, autoFlush, pageEncoding, contentType, charset etc.

Comment (k-kato) [reply WebRoot/public/login_history.jsp]: please move style block to a css file

File: WebRoot/public/login_history.jsp [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	background-color: #EEE;
+	display: table-footer-group;
+	font-weight: bold;
+}
+.divTableBody {
+	display: table-row-group;
+}
+</style>
+<div align="center">
+
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: When the page is called via jsp:forward, the jsp page should have html, (head,) and body tags.

File: WebRoot/public/login_history.jsp [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+}
+.divTable{
+	display: table;
+	width: 50%;
+}
+.divTableRow {
+	background-color: #F2F2F2;
+	display: table-row;
+}
+.divTableHeading {
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: Please remove it if it is not used.

File: WebRoot/public/login_history.jsp [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+	background-color: #EEE;
+	display: table-header-group;
+}
+.divTableCell, .divTableHead {
+	border: 1px solid #999999;
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: Please remove divTableHead if it is not used.

File: WebRoot/public/login_history.jsp [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+	display: table-cell;
+	padding: 3px 10px;
+}
+.divTableHeading {
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: It is defined twice (line 16 and 25). Please merge it, or remove it if it is not used

File: WebRoot/public/login_history.jsp [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+	background-color: #EEE;
+	display: table-header-group;
+	font-weight: bold;
+}
+.divTableFoot {
```
Comment (k-kato) [inline WebRoot/public/login_history.jsp]: - Please remove it if it is not used

File: WebRoot/public/login_history.jsp (hunk 1 above)
Comment (k-kato) [reply WebRoot/public/login_history.jsp]: Please add license block

File: WebRoot/public/login_history.jsp [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    <title><fmt:message key="zimbraLoginHistory"/></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="<fmt:message key="zimbraLoginMetaDesc"/>">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+    <link rel="stylesheet" type="text/css" href="<c:url value='/css/login-history.css'>
+        <c:if test="${not empty param.debug}">
+            <c:param name="debug" value="${param.debug}" />
+        </c:if>
+    </c:url>">
```
Comment (silentsakky) [inline WebRoot/public/login_history.jsp]: where is start tag for this?

File: WebRoot/public/login_history.jsp [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+-->
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+    <title><fmt:message key="zimbraLoginHistory"/></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="<fmt:message key="zimbraLoginMetaDesc"/>">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+    <link rel="stylesheet" type="text/css" href="<c:url value='/css/login-history.css'>
+        <c:if test="${not empty param.debug}">
+            <c:param name="debug" value="${param.debug}" />
```
Comment (silentsakky) [inline WebRoot/public/login_history.jsp]: where are we using this param?

**Pull request merged**